src/api/register.ts

```diff
@@ -0,0 +1,24 @@
+export const postRegister = async (
+  email: string,
+  password: string,
+
+  onSuccess: (response?: string) => void,
+  onError: (errorMessage: string) => void
+) => {
+  try {
+    const response = await fetch("http://localhost:3000/register", {
```

Comment:
http://localhost:3000

this should be moved as a constant variable, so it's easy to rewire application to any other compatible api, for example to production url like https://mysampleapp.com/register

so, create /constants folder in root client folder and 

```
const baseUrl = http://localhost:3000
```

```
const response = await fetch(`${baseUrl}/register`, {})
```

Reply:
ok, done